src/openloco/game_commands.h

```diff
@@ -39,6 +51,15 @@ namespace openloco::game_commands
         do_command(71, regs);
     }
 
+    inline bool do_50(uint8_t bl, uint8_t townId)
```

Comment:
```suggestion
    inline bool do_50(uint8_t bl, town_id_t townId)
```

Reply:
I agree, but as I said in another comment:

> `town_id_t` is defined in `town.h`. We should probably consider moving these types to a more common location so we don't have to include the entire header (and its dependencies) for one little type.
> 
> … Preferably in a different PR.

I think we should move a bunch such types to e.g. `types.hpp` for easy inclusion.